modules/beta-private-cluster/cluster_regional.tf

```diff
@@ -173,6 +173,10 @@ resource "google_container_cluster" "primary" {
       state    = database_encryption.value.state
     }
   }
+
+  workload_identity_config {
```

Comment:
Changes should be made in the `autogen/` folder, not directly in the `modules/` themselves.

Reply:
Changed moved to autogen folder